server/server_test.go

```diff
@@ -430,17 +424,17 @@ func TestDisconnectingServer(t *testing.T) {
 		return
 	}
 
-	time.Sleep(500 * time.Millisecond)
+	time.Sleep(idleTimeout)
```

Comment:
We are now basically sleeping the same time we set for the timeout (plus the extra time until the processor picks up this thread). Is this the intention of the change?